src/validation.cpp

```diff
@@ -1379,6 +1570,7 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptPackage(const Package& package,
                 // future.  Continue individually validating the rest of the transactions, because
                 // some of them may still be valid.
                 quit_early = true;
+                package_state.Invalid(PackageValidationResult::PCKG_TX, "transaction failed");
```

Comment:
```suggestion
                results.emplace(wtxid, single_res);
                package_state.Invalid(PackageValidationResult::PCKG_TX, "transaction failed");
```

Reply:
Yes good point, I think this is correct.